autogpt/speech/macos_tts.py

```diff
@@ -1,21 +1,59 @@
 """ MacOS TTS Voice. """
 import os
+import subprocess
 
+from autogpt.config import Config
 from autogpt.speech.base import VoiceBase
 
-
 class MacOSTTS(VoiceBase):
     """MacOS TTS Voice."""
 
     def _setup(self) -> None:
-        pass
+        """Set up the voices for MacOSTTS.
+
+        Returns:
+            None: None
+        """
+        cfg = Config()
+        available_voices = self.get_voice_names()
+        default_voices = ["Alex", "Victoria"]
```

Comment:
As default first voice I would set it to use say without -v which will be the default of the system. 
i found that using that gives the best voice quality

Reply:
Those are the default voices for extra agents.

The default voice is not using `-v`

```
        if voice_index == 0:
            os.system(f'say "{text}"')
```

Reply:
correct

Reply:
AH seems I misread, nice!